llvm/utils/TableGen/X86DisassemblerTables.cpp

```diff
@@ -926,7 +951,9 @@ void DisassemblerTables::emitContextTable(raw_ostream &o, unsigned &i) const {
       else
         o << "IC_VEX";
 
-      if ((index & ATTR_EVEX) && (index & ATTR_EVEXL2))
+      if ((index & ATTR_EVEXB) && (index & ATTR_EVEXU))
+        ; // Ignore ATTR_VEXL and ATTR_EVEXL2 under YMM rounding.
```

Comment:
Do you mean `L`, `L2` bits are ignored if EVEX.U and EVEX.b are set?  If so, should we add decoding test for it?

Reply:
`L` and `L2` are repurposed as rouding bits, they have been test with {sae-*} tests.